From db19dfa17261c3d04de0202b2809ba8abb70326b Mon Sep 17 00:00:00 2001
From: Davanum Srinivas <davanum@gmail.com>
Date: Thu, 2 Jul 2015 05:55:13 -0400
Subject: [PATCH] Fix build break with Fixtures 1.3

Our explicit call to cleanUp messes things up in latest
fixture, so we need to call _clear_cleanups to stop
the test from breaking

Change-Id: I8ce2309a94736b47fb347f37ab4027857e19c8a8
---
 tests/unit/test_moxstubout.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/unit/test_moxstubout.py b/tests/unit/test_moxstubout.py
index 03b2529..8a12705 100644
--- a/tests/unit/test_moxstubout.py
+++ b/tests/unit/test_moxstubout.py
@@ -30,3 +30,4 @@ def test_basic_stubout(self):
         f.cleanUp()
         after2 = TestMoxStubout._stubable
         self.assertEqual(before, after2)
+        f._clear_cleanups()
